docs/about/overview/encryption/limitations.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: limitations
+title: Known Limitations of Meshtastic's Encryption
```

Comment:
Structurally I think it makes sense to combine this with the main encryption page. If not, I would at a minimum suggest we update the title and label to represent that there is a "Future" section in here as well.